### PR TITLE
Reverts goonchem, ports baymed, and adds physdesc feature

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -72,12 +72,12 @@
 
 
 	if (getBrainLoss() >= 60 && stat != DEAD)
-		if (prob(3))
+		if (prob(100))
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/life.dm]: revert this line or no.

**Comment by optimumtact** [comment]
They're probably a little too fourth wall breaking but :+1: for effort

**Comment by Xhuis** [comment]
>complains about fourth wall
>"lifelike texture ;_;"

yeah ok

**Comment by Iamgoofball** [comment]
:+1:

**Comment by RemieRichards** [comment]
Add "Shiggy Diggy"

Add it now.

**Comment by GunHog** [comment]
PORT GOOFMED!!!!

**Comment by MrPerson** [comment]
Just because there are already existing ones that have a trait is not a good reason to add more, assuming that trait is something we don't like.

**Comment by Fox-McCloud** [comment]
"closd for merbegging"

lol...

**Comment by ike709** [comment]
>was hoping for baymed

gibber for the gibber gods

**Comment by Miauw62** [comment]
will "ayy lmao" and "darood- sansorm" ever be in the retardspeak? find out next time in Space Drama!

**Comment by vista-** [comment]
make the spelling less atrocious, the point of the retarded gibbering is the content, not the presentation

**Comment by Xhuis** [comment]
> without oxigen blob don't evoluate?

whatever you say

**Comment by vista-** [comment]
I meant the ones that are being added. Just make the spelling a bit less atrocious, just to the point where you can comprehend them without having to read them twice.

**Comment by Alek2ander** [comment]
>this took 6 commits

**Comment by Xhuis** [comment]
Squashed commits and added a new phrase

**Comment by Amnestik** [comment]
I don't like that these imply to players that it's okay to make OOC references IC when you'll get banned for it for 15 minutes nowadays. Unless it's only acronyms they ban for, I'm not sure.

**Comment by optimumtact** [comment]
Act like a retard, get treated like a retard
:smile:

**Comment by Amnestik** [comment]
Characters don't get banned for acting like retards, though?

**Comment by Cheridan** [comment]
Yeah I don't really like characters referencing admins IC even if it is literally retarded

**Comment by Xhuis** [comment]
There is no direct references to players/admins that I see... if there is, let me know and I'll cut them

**Comment by Cheridan** [comment]
i meant "syebl is down ADMENS PLEAS" and "pray can u MAke me admens"

**Comment by Amnestik** [comment]
Those aren't very convincing typos either




I don't think anyone would think to spell it admen unless they were being ironic

**Comment by Miauw62** [comment]
how about "its down panic1!!!!!!!"

**Comment by Razharas** [comment]
Perfect

**Pull request merged**